### ISSUE TITLE
URL should be case insensitive

### ISSUE THREAD

**Issue by neilhwatson** [chubin/cheat.sh#18]
cheat.sh/perl/subroutines

and

    cheat.sh/Perl/Subroutines

Should return the same thing.

**Comment by steils** [comment]
@chubin Still actual.

**Comment by abitrolly** [comment]
https://cheat.sh/Perl/Subroutines now gives Internal Server Error. @chubin something is broken.

**Comment by chubin** [comment]
@abitrolly Should work now, but the issue is still actual. I agree with the proposal. The queries should be case-insensitive

**Comment by abitrolly** [comment]
The fix is to add `query = query.lower()` somewhere around these lines.

https://github.com/chubin/cheat.sh/blob/7f769d6f3697541e55fd3ea9b71f190296529e48/lib/cheat_wrapper.py#L90-L91

Or maybe after the query is parsed if `search_options` are still case-sensitive at this point.

**Comment by chubin** [comment]
Yes, but this change is not enough, because filenames are case insensitive. It is possible to make them lower case in our repositories, but we can't influence external repositories, so more changes needed, but it is not a problem
